ow_lander/scripts/ground_detection.py

```diff
@@ -108,10 +124,12 @@ def _tf_lookup_position(self, timeout=rospy.Duration(0.0)):
     return t.transform.translation
 
   def _tf_2_method(self):
+
     return self._check_condition(self._tf_lookup_position())
 
   @property
   def ground_position(self):
+
```

Comment:
Why these new lines being added?
```suggestion

```

Reply:
Some methods had empty lines after the definition lines, some others didn't. So I added an empty line to the ones that didn't have it for consistency. I'll remove them.